### DISCUSSION TITLE
Moving portal content and config between pimcore environments

### DISCUSSION THREAD

**Discussion by phosphordev** [pimcore/pimcore#17206] [Q&A / Support] [answered]
Is there a supported or recommended method to move portal engine content and config from one environment to another?

E.g dev, staging, prod server configurations

Editor creates/alters pages in staging and needs to move the changes to production. What is the process for this with Pimcore and Portal Engine?

**Comment by markus-moser** [comment, marked as answer]
Unfortunately there is no way for doing this. The complex thing here is that there are multiple entities (document, asset, data object) involved and as the `id` columns of those entities are auto increment values you would need to map all ids and somehow ensure that all entities get synced. This is a complex thing and therefore not easily doable. The only realistic way for now is doing this manually.

**Reply by phosphordev** [reply]
Thanks @markus-moser  , I thought this might be the case but wanted to confirm we had not missed something.
